botocore/utils.py

```diff
@@ -389,6 +389,20 @@ def parse_to_aware_datetime(value):
     return datetime_obj
 
 
+def datetime2timestamp(dt, default_timezone=tzutc()):
```

Comment:
Is tzutc() mutable? If it is, I rather the have default be ``None`` and set the value to ``tzutc()`` in the function if it is ``None``.